packages/client/src/toHexString.ts

```diff
@@ -0,0 +1,5 @@
+import { utils } from "ethers";
+
+export const toHexString = (data: string) => {
```

Comment:
We could add unit tests for these types of standalone helper functions in the near future.

Reply:
https://github.com/zeta-chain/toolkit/issues/232